content/en/blog/_posts/2025-xx-xx-pod-level-resources/index.md

```diff
@@ -0,0 +1,179 @@
+---
+layout: blog
+title: "Kubernetes v1.34: Pod Level Resources Graduated to Beta"
+date: 2025-xx-xx
+draft: true
+slug: kubernetes-v1-34-pod-level-resources
+author: Dixita Narang (Google)
+---
+
+On behalf of the Kubernetes community, I am thrilled to announce that the Pod Level Resources feature has graduated to Beta in the Kubernetes v1.34 release and is enabled by default! This significant milestone introduces a new layer of flexibility for defining and managing resource allocation for your Pods. This flexibility stems from the ability to specify CPU and memory resources for the Pod as a whole. Pod level resources can be combined with the container-level specifications to express the exact resource requirements and limits your application needs.
+
+#### What are Pod Level Resources?
+
+Until recently, resource specifications that applied to Pods were primarily defined
+at the individual container level. While effective, this approach sometimes required
+duplicating or meticulously calculating resource needs across multiple containers
+within a single Pod. As a beta feature, Kubernetes allows you to specify the CPU,
+memory and hugepages resources at the Pod-level. This means you can now define
+resource requests and limits for an entire Pod, enabling easier resource sharing
+without requiring granular, per-container management of these resources where
+it's not needed.
+
+
+#### Why Does Pod Level Resources Matter?
+
+This feature enhances resource management in Kubernetes by offering *flexible resource management* at both the Pod and container levels.
+
+* It provides a consolidated approach to resource declaration, reducing the need for
+  meticulous, per-container micromanagement, especially for Pods with multiple
+  containers. 
+* Pod-level resources enable containers within a pod to share unused resoures
+  amongst themselves, promoting efficient utilization within the pod. For example, it prevents sidecar containers from becoming performance bottlenecks. Previously, a sidecar (e.g., a logging agent or service mesh proxy) hitting its individual CPU limit could be throttled and slow down the entire Pod, even if the main application container had plenty of spare CPU. With pod-level resources, the sidecar can borrow unused resources from the Pod's shared budget, ensuring smooth operation during traffic spikes.
```

Comment:
```suggestion
  amongst themselves, promoting efficient utilization within the pod. For example, it prevents sidecar containers from becoming performance bottlenecks. Previously, a sidecar (e.g., a service mesh proxy) hitting its individual CPU limit could be throttled and slow down the entire Pod, even if the main application container had plenty of spare CPU. With pod-level resources, the sidecar can borrow unused resources from the Pod's shared budget, ensuring smooth operation during traffic spikes.
```

Reply:
this is not resolved. The paragraph emphasizes sharing, while I think it is more important to emphasize that the limit will be applied uniformely at the same time to both - sidecar and a main container